### DISCUSSION TITLE
Visitor online

### DISCUSSION THREAD

**Discussion by mirmoh3n** [chatwoot/chatwoot#10157] [Feature Requests]
I'm also waiting for the Live Visitor feature like on tawkto,
I'm used to using tawkto and monitoring visitor traffic, when using chatwoot I feel strange because I'm not used to it and have to daydream without seeing the traffic that is happening on the website, in tawkto I can monitor the visitor currently accessing the page.
hopefully this feature will be available soon

**Comment by slade991** [comment]
Waiting on that as well before we switch.
Is it planned ?
